test/build-test/prepublish/lib/build-spec.js

```diff
@@ -0,0 +1,37 @@
+const sinon = require( 'sinon' );
+const chai = require( 'chai' );
+const expect = chai.expect;
+const sinonChai = require( 'sinon-chai' );
+chai.use( sinonChai );
+const proc = require( 'child-process-promise' );
+const path = require( 'path' );
+const rootDir = require( '../root-path' );
```

Comment:
Why not `rootPath`? When the name is so similar like this, I'd just make it the same as what's imported.

Reply:
Good catch, left over from some optimization I made and renamed things.